lualib/resty/kong/tls.lua

```diff
@@ -59,6 +61,7 @@ if subsystem == "http" then
         int depth);
     int ngx_http_lua_kong_ffi_get_socket_ssl(ngx_http_lua_socket_tcp_upstream_t *u,
         void **ssl_conn);
+    int ngx_http_lua_ffi_ssl_disable_http2_alpn(ngx_http_request_t *r, char **err);
```

Comment:
remove ssl_ and name it `ngx_http_lua_ffi_disable_http2_alpn` to be consistent with others